lib/browser/api/app.js

```diff
@@ -17,6 +18,12 @@ let dockMenu = null
 Object.setPrototypeOf(App.prototype, EventEmitter.prototype)
 EventEmitter.call(app)
 
+const castArgs = function (args) {
+  return args.map((arg) => {
+    return typeof arg !== 'string' ? `${arg}` : arg
```

Comment:
Why not use a `String()` call to convert to string? Calling `String` without `new` will produce a primitive not a `String` instance.

Reply:
done